system/jlib/jatomic.hpp

```diff
@@ -23,8 +23,15 @@
 #ifdef _WIN32
 inline static void spinPause() { YieldProcessor(); }
 #else
-#include "xmmintrin.h"
+# include "x86intrin.h"
+# if defined(_ARCH_X86_)
```

Comment:
We dont seem to have a generic intel/amd/x86* macro for the platform to put the if within, but we could do this -

\\#if defined(_ARCH_X86_64_) || defined(_ARCH_X86_)
  \\#include "x86intrin.h"
  \\#if defined(_ARCH_X86_)
  inline static void spinPause() { __pause(); }
  \\#else
  inline static void spinPause() { _mm_pause(); }
  \\#endif
\\#endif